lib/managers/web-links.js

```diff
@@ -86,4 +86,35 @@ WebLinks.prototype.get = function(weblinkID, qs, callback) {
 	this.client.get(apiPath, params, this.client.defaultResponseHandler(callback));
 };
 
+/**
+ * Updates information for a web link.
+ *
+ * API Endpoint: '/web_links/:weblinkID'
+ * Method: PUT
+ *
+ * @param {string} weblinkID - The Box ID of the web link being updated
+ * @param {?Object} options - Additional parameters
+ * @param {string} [options.name] - Name for the web link. Will default to the URL if empty.
+ * @param {string} [options.description] - Description of the web link. Will provide more context to users about the web link.
+ * @param {Function} callback - Passed the updated web-link information if it was acquired successfully, error otherwise
+ * @returns {void}
+ */
+WebLinks.prototype.update = function(weblinkID, options, callback) {
+	var apiPath = urlPath(BASE_PATH, weblinkID),
+		params = {
+			body: {
+			}
+		};
+
+	if (options && options.name) {
+		params.body.name = options.name;
+	}
+
+	if (options && options.description) {
+		params.body.description = options.description;
+	}
```

Comment:
These should be updated to the standard options passing style, where all options are passed in the body